### PR TITLE
fix: #10232 Improved italian translations

### LINKED ISSUE #10232: Italian translation
Please can you translate all the menù on the right click?
Now he is half Italian and half English, a bit annoying.

Thank you,
Simone
dbeaver

### PR DESCRIPTION
**Pull request by acoluzzi**
I've tried to fill in as much tokens as I could while resolving #10232

### CODE REVIEW

File: plugins/org.jkiss.dbeaver.ui.editors.data/OSGI-INF/l10n/bundle_it.properties [hunk 1]
```diff
@@ -37,4 +37,41 @@ page.org.jkiss.dbeaver.preferences.main.resultset.presentation.name = Aspetto
...
 page.org.jkiss.dbeaver.preferences.main.dataviewer.name = Dizionari
+command.org.jkiss.dbeaver.core.resultset.openWith.name=Apri con
+command.org.jkiss.dbeaver.core.resultset.cell.setNull.name=Imposta a NULL
+command.org.jkiss.dbeaver.core.resultset.cell.setNull.description=Imposta il valore delle celle a NULL
+command.org.jkiss.dbeaver.core.resultset.cell.setDefault.name=Imposta valore default
+command.org.jkiss.dbeaver.core.resultset.cell.setDefault.description=Imposta la cella al valore di default
+command.org.jkiss.dbeaver.core.resultset.export.name=Esporta dati
+command.org.jkiss.dbeaver.core.resultset.export.description=Esporta result set su file o altra tabella
+command.org.jkiss.dbeaver.core.resultset.filterMenu.distinct.name=Filtra per valore
+command.org.jkiss.dbeaver.ui.editors.data.forSelection.name=MOstra dati
```
Comment (LonwoLonwo) [inline plugins/org.jkiss.dbeaver.ui.editors.data/OSGI-INF/l10n/bundle_it.properties]:
```suggestion
command.org.jkiss.dbeaver.ui.editors.data.forSelection.name=Mostra dati
```

**Comment by LonwoLonwo** [comment]
Thanks for your contribution.
Looks nice.
Waiting for one more review and QA Team.

**Review by LonwoLonwo — APPROVED** [review]

**Review by ShadelessFox — APPROVED** [review]

**Comment by acoluzzi** [comment]
Hi guys, Can this be merged by any case?

**Comment by ShadelessFox** [comment]
This PR awaits our QA team.

**Comment by MashaKorax** [comment]
verified

**Pull request merged**